app/pods/bucket/model.js

```diff
@@ -104,6 +104,11 @@ var Bucket = DS.Model.extend({
      * @type Boolean
      */
     isActive: function() {
+        if(this.get('bucketTypeId') === 'default') {
+            // Buckets in the Default type don't have the 'active' attribute
+            // in the props, but are actually active.
+            return true;
+        }
```

Comment:
So what happens if the default bucket type is inactive? Is the active attribute then passed down? If so wouldn't it be easier to just set the property to have a default value of `true` and not put this here?